### PR TITLE
Comp avatars

### PR DESCRIPTION
**Pull request by jeetsdev**
Added avatar components.

### CODE REVIEW

**Review by NtshVrm — APPROVED** [review]

**Review by SK02K1 — APPROVED** [review]

**Review by manismk — APPROVED** [review]
Looks good

**Review by ayush65 — APPROVED** [review]

File: css/docs.css [hunk 1]
```diff
@@ -5,27 +5,29 @@ flex-direction: column;
...
     color: var(--text-sec);
     position: relative;
+    justify-content: space-between;
 }
 
 /** Sidebar style here */
 
 .sidebar{
-    box-shadow: 0px 5px 10px 0px var(--color-pri);
+    box-shadow: 0px 5px 10px 0px var(--bg-color-sec);
```
Comment (MaruthiPothuganti) [inline css/docs.css]: Box shadow can also be done box-shadow: 5px 10px #888888; for right and down side shadows

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended change in box-shadow property

File: css/docs.css (hunk 1 above)
Comment (jeetsdev) [reply css/docs.css]: Refactored that, but I think it's imp to use x-axis and y-axis values in my situation.

**Pull request merged**